### PR TITLE
Import HTTP error exception

### PR DESCRIPTION
**Pull request by shadowhand**
Refs https://github.com/middlewares/payload/issues/5#issuecomment-397416217

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/) 
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [2.0.0] - ???
```
Comment (oscarotero) [inline CHANGELOG.md]: This version contains only an addition, there's no breaking changes, so I think `1.1.0` is more accurate

**Review by oscarotero — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (shadowhand) [reply CHANGELOG.md]: Fixed.

**Review by oscarotero — APPROVED** [review]

**Comment by shadowhand** [comment]
@oscarotero are you going to tag this or should I?

**Comment by oscarotero** [comment]
I'm tagging right now

**Comment by oscarotero** [comment]
Done!

**Pull request merged**